### DISCUSSION TITLE
Add Setting to ask for confirmation when performing Org changing actions (Delete/Deploy)

### DISCUSSION THREAD

**Discussion by valerioa-amd** [forcedotcom/salesforcedx-vscode#5455] [Ideas]
**Is your feature request related to a problem? Please describe.**
When right-clicking on an object, Delete/Deploy/Diff are very close and the wrong action could be clicked by mistake

**Describe the solution you'd like**
A setting to enable/disable a pop-up asking "Are you sure to perform Delete/Deploy? The action is not reversible!"

**Additional context**
![image](https://github.com/forcedotcom/salesforcedx-vscode/assets/146344717/539aa4e5-5544-419c-b2e4-d39ba958e615)

**Comment by daphne-sfdc** [comment]
Thank you @valerioa-amd for your suggestion.  We'll let you know if we implement this feature.
